### DISCUSSION TITLE
RaspberryMatic 3.69.4 Nightly Snapshot

### DISCUSSION THREAD

**Discussion by ghost** [OpenCCU/OpenCCU#2268] [Development]
Alles funktioniert seit mehr als 10 Stunden hervorragend.
Herzlichen Glückwunsch ;-)
Danke!

P.S. Baxxy13 Ich würde dir auch ein Bier ausgeben, wenn ich die Möglichkeit und Gelegenheit dazu hätte hier. 
![1](https://user-images.githubusercontent.com/81189856/226962976-ef10a1da-27e5-4560-84a6-552ba21d2f6b.jpg)
![33](https://user-images.githubusercontent.com/81189856/226972981-c59396a5-d14d-4848-86de-577dc2b1affd.jpg)

**Comment by jens-maus** [comment]
Danke für das Feedback. Allerdings möchte ich trotzdem nochmal explizit darauf hinweisen das es hier um einen nightly snapshot bzw. development build geht. 3.69.4 ist immer noch nicht offiziell released von eQ3. Deshalb installation auf eigene Gefahr und Risiko!
